package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "math-magicians",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.1.2"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  }
```

Comment:
- [ ] There should be `devDependencies` in your `package.json`, unless you made some mistake in your `react app` initialization. As a result, your app is not running correctly. Kindly fix this error and submit it for another review. :pray: 

![image](https://user-images.githubusercontent.com/64894927/128779557-f09849ae-5885-4f16-83bd-9322848cd27d.png)

Reply:
Hi Henok. I tested again by cloning the project in another folder and It's running ok.